apps/_example05app/utils/excal.utils.js

```diff
@@ -0,0 +1,285 @@
+function isValidEmail (email) {
```

Comment:
File is called `excal`, why not `excel` ?

Reply:
mistake %)